addons/rose/addon/styles/rose/components/table/_table.scss

```diff
@@ -35,7 +39,33 @@
     }
   }
 
+  // Grow to full available width
+  .rose-table-cell-expand {
+    width: 100%;
+
+    & > * {
+      vertical-align: inherit;
+      margin: 0;
+    }
+  }
+
   [class*='rose-button'] {
     margin-bottom: 0;
   }
+
+  &.rose-table-condensed {
+    .rose-table-cell {
+      @include type.type(s);
```

Comment:
Is the type actually small in the designs?

Reply:
`condensed` styles have been extracted from existing key/value component styles as one of the goals on this PR was to make key/value component obsolete.

Reply:
I doubled checked and the designs use 14px or "small" so this should be fine.